frontend/src/component/project/Project/ProjectForm/ProjectForm.tsx

```diff
@@ -190,7 +236,9 @@ const ProjectForm: React.FC<IProjectForm> = ({
                     onChange={e => setProjectName(e.target.value)}
                     error={Boolean(errors.name)}
                     errorText={errors.name}
-                    onFocus={() => clearErrors()}
+                    onFocus={() => {
+                        delete errors.name;
+                    }}
```

Comment:
previously, focusing the project name would clear all errors, including the ones for the naming pattern. this is a bit overzealous, so we'll only clear the errors.name property here.